File: Random_Files/USconverter.py
import math
feetToInches = lambda feet: feet*12
def inchesToFeet(inches):
    return inches / 12
def inchesToInchesAndFeet(inches):
    return {
        "Feet":math.floor(inches/12),
        "Inches":inches%12
    }
print(feetToInches(7))
print(inchesToFeet(156))
print(inchesToInchesAndFeet(157))

File: Random_Files/USconverter.js
let feetToInches = (feet) => feet*12;
let inchesToFeet = (inches) => {
    return inches / 12;
};
function inchesToInchesAndFeet(inches){
    return {
        "Feet":Math.floor(inches/12),
        "Inches":inches%12
    };
}
console.log(`7 Feet is ${feetToInches(7)} inches`);
console.log(`156 inchest is ${inchesToFeet(156)} feet`);
console.log(inchesToInchesAndFeet(157));
